module_11/index.php

```diff
@@ -0,0 +1,124 @@
+    </ul>
+</div>
+
+<table width="100%" border="0" cellspacing="0" cellpadding="0">
+    <tr>
+        <td class="left-collum-index">
+
+            <h1>Возможности проекта —</h1>
+            <p>Вести свои личные списки, например покупки в магазине, цели,
+                задачи и многое другое. Делится списками с друзьями и
+                просматривать списки друзей.</p>
+
+
+        </td>
+        <td class="right-collum-index">
+
+            <div class="project-folders-menu">
+                <ul class="project-folders-v">
+
+                    <li <?php
+                    if (array_key_exists("login", $_GET) == true
+                        && $_GET['login'] == 'yes'
+                    ) { ?> class="project-folders-v-active" <?php
+                    } ?>><a href="/?login=yes">Авторизация</a></li>
+                    <li><a href="#">Регистрация</a></li>
+                    <li><a href="#">Забыли пароль?</a></li>
+                </ul>
+                <div class="clearfix"></div>
+            </div>
+
+            <?php
+            if (array_key_exists("login", $_GET) == true
+                && $_GET['login'] == 'yes'
+            ) { ?>
+                <div class="index-auth">
+                    <form action="/?login=yes" method="post">
+                        <table width="100%" border="0" cellspacing="0"
+                               cellpadding="0">
+                            <tr>
+                                <td class="iat">
+                                    <label for="login_id">Ваш e-mail:</label>
+                                    <input id="login_id" size="30" name="login">
+                                </td>
+                            </tr>
+                            <tr>
+                                <td class="iat">
+                                    <label for="password_id">Ваш пароль:</label>
+                                    <input id="password_id" size="30"
+                                           name="password" type="password">
+                                    <?php
+                                    if (array_key_exists('login', $_POST)) {
+                                        for ($i = 0; $i < count($users); $i++) {
+                                            if ($_POST['login'] === $users[$i]
+                                                && $_POST['password']
+                                                === $passwords[$i]
+                                            ) {
+                                                include "include/success_message.php";
+                                                break;
+                                            } else {
+                                                include "include/error_message.php";
+                                                break;
+                                            }
+                                        }
+                                    } ?>
```

Comment:
Create file `authorization.php` and place all auth logic there.
Include that file at the beginning  of the `index.php` file. In the file with logic you should create variable with condition of your authtorization. After you will use that variable for detemining your auth status and showing all necessary fields.